docs/source/getting_started.rst

```diff
@@ -3,16 +3,51 @@
 Getting Started
 ***************
 
-A conda (or pip) installation is still not possible. Meanwhile,
-you need to clone (or fork) the repository.
+Model Database Access
+---------------------
+
+Simulation model parameters are stored in a MongoDB-type database.
+Many gammasim-tools applications depend on access to this database; ask one of the developers for the credentials.
+
+Credentials for database access are passed on to gammasim-tools applications using environmental variables.
+For database access, copy \
+`set_DB_environ_template.sh <https://github.com/gammasim/gammasim-tools/blob/master/set_DB_environ_template.sh>`_ to a new file named ``set_DB_environ.sh``, and update it with the credentials:
+
+.. code-block::
+
+    export DB_API_USER=<db_user_name>
+    export DB_API_PW=<db_password>
+    export DB_API_PORT=<db_port>
+    export DB_SERVER=<db_server>
+
```

Comment:
I am also using locally the SIMTEL_PATH in the same file. I believe we, the developers, could think in boxes and separate it, but in my opinion the user should get one configuration and be ready to go. It also doesnt hurt to have it defined and maybe not use it (just in case).

Reply:
Simulation software is really a different thing compared to the database setup. It might also change in future (e.g., if we put corsika/sim_telarray into containers) or if we use other simulation software. Let's keep it separate.